### PR TITLE
RANCHER-739 Reconfigure Jenkins jobs related to the process of update of the modules.

### PR DESCRIPTION
**Pull request by OHaimanov**
Jira: https://issues.folio.org/browse/RANCHER-739

### CODE REVIEW

File: src/org/folio/rest_v2/Authorization.groovy [hunk 1]
```diff
@@ -149,4 +149,41 @@ class Authorization extends Common {
...
+     * Checks if the tenant is locked by trying to login with the tenant's admin credentials.
+     *
+     * @param tenant The OkapiTenant object representing the tenant.
+     * @return Returns 'true' if the tenant is locked, otherwise returns 'false'.
+     *         If the HTTP response code is other than 404 (NOT FOUND), an exception will be thrown.
+     * @throws RequestException when there's an unexpected error during the REST call.
+     */
+    boolean isTenantLocked(OkapiTenant tenant) {
+        // Construct the URL for the login endpoint.
+        String url = generateUrl("/authn/login")
```
Comment (vkapylou) [inline src/org/folio/rest_v2/Authorization.groovy:163]: As a new feature with RTR will be in place, should we consider using "/authn/login-with-expiry"?
Related doc: https://wiki.folio.org/pages/viewpage.action?pageId=96414255

Comment (OHaimanov) [reply src/org/folio/rest_v2/Authorization.groovy:163]: I think we should created additional ticket to implement RTR feature

File: src/org/folio/rest_v2/Main.groovy [hunk 2]
```diff
@@ -80,15 +90,25 @@ class Main extends Okapi {
...
+    void update(Map<String, OkapiTenant> tenants) {
         tenants.each { tenantId, tenant ->
             tenantInstall(tenant, tenant.modules.generateInstallJsonFromIds(['okapi'], 'enable'))
             tenantInstall(tenant, tenant.modules.installJson, 900000)
-            configureTenant(tenant)
+            if (tenant.getAdminUser()) {
+                loginUser(tenant)
+                permissions.purgeDeprecatedPermissions(tenant)
+                permissions.refreshAdminPermissions(tenant, tenant.getAdminUser())
+//                configureTenant(tenant)
```
Comment (vkapylou) [inline src/org/folio/rest_v2/Main.groovy:101]: I’ll point it out just in case this shouldn’t be here 👀

Comment (OHaimanov) [reply src/org/folio/rest_v2/Main.groovy:101]: It should but need some adjustments

**Pull request merged**